x-pack/plugins/data_enhanced/server/search/es_search_strategy.ts

```diff
@@ -69,7 +69,7 @@ async function asyncSearch(
   const path = encodeURI(request.id ? `/_async_search/${request.id}` : `/${index}/_async_search`);
 
   // Wait up to 1s for the response to return
-  const query = toSnakeCase({ waitForCompletionTimeout: '1s', ...queryParams });
```

Comment:
I think that the issue with the tests should be resolved by retrying in the tests, not increasing the initial waitForCompletionTimeout. Isn't that so?

Reply:
My last commit was to test if an increase of the waitForCompletionTimeout solves the flakiness of the tests, it does:

https://kibana-ci.elastic.co/job/kibana+flaky-test-suite-runner/378/

so there are 2 approaches her to solve this: increase the timeout oder retry the test

Reply:
@lukasolson @lizozom @lukasolson Question is, why the user, or in this case the test is getting the message, that there're no results matching this criteria. In this case there are, but it took longer than the `waitForCompletionTimeout`, shouldn't it continue searching in this case with GET async search? If the system is for some reason slower, that's what happening her, it shouldn't feedback that there're no result.

Reply:
This is what I get when I start an expensive search in my 7.7 staging cluster, wildcard search over 50mio records:

<img width="332" alt="Bildschirmfoto 2020-04-20 um 11 55 28" src="https://user-images.githubusercontent.com/463851/79739230-30c8ee00-82fe-11ea-88f9-5a0f79ece172.png">

It did continue while I was writing it, and I suddenly got this, shouldn't there be a message, that it timed out? 

<img width="1268" alt="Bildschirmfoto 2020-04-20 um 11 59 00" src="https://user-images.githubusercontent.com/463851/79739361-5fdf5f80-82fe-11ea-9414-4359f975a5c5.png">